tests/test_plugin.py

```diff
@@ -1,7 +1,6 @@
 # -*- coding: utf-8 -*-
 import pytest
-from pluggy.manager import PluginManager
-
+from pluggy.manager import PluginManager  # pylint: disable=unused-import
```

Comment:
Do we actually need this and the commented code block in  `test_pytest_vcr_incompatibility` ?